django_select2/forms.py

```diff
@@ -349,14 +362,19 @@ def set_to_cache(self):
             'search_fields': self.search_fields,
             'max_results': self.max_results,
             'url': self.get_url(),
+            'dependent_fields': self.dependent_fields,
         })
 
-    def filter_queryset(self, term, queryset=None):
+    def filter_queryset(self, term, queryset=None, **dependent_fields):
         """
         Return QuerySet filtered by search_fields matching the passed term.
 
         Args:
             term (str): Search term
+            queryset (django.db.models.QuerySet): QuerySet to select choices from.
```

Comment:
> Ok, than lets finish this up. Please make sure to fix the doc test. Queryset (capital Q) should work.

You are talking about this one?
This is the parameter of a function, should we change case?